tests/seeding/test_find_or_create.py

```diff
@@ -267,8 +266,8 @@ def test_find_or_create_raise_error_project_exist_multiple(project_collection):
 
 def test_find_or_create_project_no_team(project_collection):
     # test when project collection has no team
-    collection = project_collection(team_id=None)
-    with pytest.raises(NotImplementedError):
```

Comment:
@kroenlein find_or_create relies in first enumerating the datasets on the dataset_collection, as intentionally the team_id wasn't provided, it is now going to fail in a different place.

Reply:
I think no-team mode doesn't even work with the platform anymore.  If this is the case, this should be throwing errors.

Reply:
It will, some errors will be thrown in different places given the heavy dependency of teams in the dataset collection.